language/move-lang/src/parser/syntax.rs

```diff
@@ -1816,7 +1877,7 @@ fn parse_module(tokens: &mut Lexer<'_>) -> Result<ModuleDefinition, Error> {
 //              <ConstantDecl>*
 //              <MoveFunctionDecl>
```

Comment:
Please insert: \<DocComments\> \<ModuleMemberModifiers\> before \<MoveFunctionDecl\> (and per my earlier comment, I think that should change to just \<FunctionDecl\>)